DESCRIPTION

```diff
@@ -44,5 +44,7 @@ VignetteBuilder:
     knitr
 Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.1.1.9001
+RoxygenNote: 7.1.2
 Config/testthat/edition: 3
+Depends: 
+    R (>= 3.5.0)
```

Comment:
so, this is to use `tempdir(check = TRUE)`, and it's not a too recent R version, is that ok?

Reply:
What do we need it for? I'd rather avoid this if possible.